package.json

```diff
@@ -39,8 +39,7 @@
     "@atlaskit/css-reset": "^2.0.5",
     "@atlaskit/reduced-ui-pack": "^8.19.0",
     "@octokit/auth-app": "^2.6.0",
-    "@sentry/integrations": "^5.15.5",
-    "@sentry/node": "^5.15.5",
+    "@sentry/node": "^5.5.0",
```

Comment:
It looks like we are actually downgrading from 5.15.x to 5.5.x 

Should we use the latest version instead?